tests/rustdoc-ui/lints/invalid-html-tags-ice-146890.rs

```diff
@@ -0,0 +1,24 @@
+// regression test for https://github.com/rust-lang/rust/issues/146890
```

Comment:
Please also explain what it's actually testing.

Reply:
I'm not really sure what extra info you want. It's testing to make sure this html doesn't cause an ICE.  I understand what the bug was, but I don't entirely understand why it's so hard to reproduce.